### PR TITLE
upgraded to latest lineicons.

### PR DESCRIPTION
**Pull request by enjine**

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -3,9 +3,14 @@ import ReactDOM from "react-dom";
 import LineIcon from "../dist"
 class HelloMessage extends React.Component {
   render() {
-    return  <div>Hello  <LineIcon size="lg" name="handshake" /></div>;
+    return  <div>
+			Hello
+			{' '}
```
Comment (anubhavmeet) [inline example/index.js]: unexpected {' '}

File: line-icons/LineIcons.css [hunk 2]
```diff
@@ -6,8 +6,12 @@ Author: lineicons.com
 -------------------------------- */
 @font-face {
   font-family: 'LineIcons';
-  src: url('../fonts/LineIcons.eot');
-  src: url('../fonts/LineIcons.eot') format('embedded-opentype'), url('../fonts/LineIcons.woff2') format('woff2'), url('../fonts/LineIcons.woff') format('woff'), url('../fonts/LineIcons.ttf') format('truetype'), url('../fonts/LineIcons.svg') format('svg');
+  src: url('./fonts/LineIcons.eot?y2l643');
```
Comment (anubhavmeet) [inline line-icons/LineIcons.css:9]: How will this build if fonts are one folder above and not in adjacent file path.

try `npm run build` locally to get the issue

File: example/index.js (hunk 1 above)
Comment (enjine) [reply example/index.js]: pushed this change out.

File: line-icons/LineIcons.css (hunk 2 above)
Comment (enjine) [reply line-icons/LineIcons.css:9]: just re-ran everything locally as well as installing it directly from my fork. It builds and renders perfectly, including the demo page. This should be good to merge.

**Review by anubhavsrivastava — APPROVED** [review]

**Pull request merged**